src/main/java/eu/ai4work/sws/model/generic/SlidingDecisionResponse.java

```diff
@@ -0,0 +1,11 @@
+package eu.ai4work.sws.model.generic;
+
+import lombok.Data;
+import java.util.Map;
+
+@Data
+public class SlidingDecisionResponse {
+    private SlidingDecisionStatus decisionStatus;
+    private Map<String, Object> decisionResults;
```

Comment:
See other comment in Controller class, this should be renamed because we actually have only one results. Suggestion to use either of the following:
- `decisionResult`
- `decisionResultDetails`

Please also adjust the README accordingly, as it will change the JSON output.

Reply:
I used the `decisionResult` as it is easy to read.  Done in commit https://github.com/AI4WORK-Project/sliding-work-sharing/pull/2/commits/e647def318c312fd99c3c5d74d04dfca9eca2a0c